### ISSUE TITLE
Support Redux v4

### ISSUE THREAD

**Issue by Patrik-Lundqvist** [gaearon/redux-devtools#391]
There are some breaking changes in Redux v4 that needs to be adressed here.

Using the `4.0.0-beta.1` release results in the following error:
```
Uncaught TypeError: Cannot read property 'state' of undefined
    at unliftState (instrument.js:579)
```
It is caused by [this change](https://github.com/reactjs/redux/commit/e54a7f22cd3e09c9903599ae430d25018cc5d48e) which appends a random string to the initial redux action type. But [here](https://github.com/zalmoxisus/redux-devtools-instrument/blob/0aa43f1aee8847ead13001aec5b5d54ce01c7e12/src/instrument.js#L508) in the store enhancher we assume that the inital action is called ```@@redux/INIT```

**Comment by ematipico** [comment]
I can confirm, current version is not compatible with version 4.

**Comment by peleteiro** [comment]
:+1:

**Comment by olee** [comment]
I'm using this code snippet right now which makes it work again - at least a bit:
```
// tslint:disable:no-var-requires
const reduxModule = require('redux');
reduxModule.__DO_NOT_USE__ActionTypes.INIT = '@@redux/INIT';
reduxModule.__DO_NOT_USE__ActionTypes.REPLACE = '@@redux/REPLACE';
```

**Comment by tomspeak** [comment]
@olee how are you using this? Does this go in your configure store?

**Comment by alvis** [comment]
There is a PR solving this upstream problem at https://github.com/zalmoxisus/redux-devtools-instrument/pull/17. Now what we need is patience.

**Comment by zalmoxisus** [comment]
Published `redux-devtools-instrument@1.8.3`. I think we don't need to increase `redux-devtools` version as well for this.

**Comment by ajcrites** [comment]
Will this fix also apply to `redux-devtools-extension`? Is there anything we can do to use the latest changes?

**Comment by zalmoxisus** [comment]
Should be fixed in `2.15.2`.

**Comment by G-Rath** [comment]
@zalmoxisus is there going to be a fix for `redux-devtools-extension`? 

It's been just under a year since last commit to that repo; has it been depreciated or moved?

**Comment by mmakrzem** [comment]
@zalmoxisus I don't understand your comment.  I'm building an Electron application that uses Redux 4.  I have installed https://www.npmjs.com/package/electron-devtools-installer so that Electron's devTools has a Redux tab, but the tab remains empty.  To get it to populate, I needed to use redux-devtools-extension, but the version published on npm is only 2.13.7 which doesn't support redux 4.  If the published version of redux-devtools-extension gets updated in npm to the latest version (https://github.com/zalmoxisus/redux-devtools-extension/releases) then my problem would be solved I think.  See https://github.com/zalmoxisus/redux-devtools-extension/issues/632
